Plant-for-the-Planet-org/planet-webapp
# Feature/app donation support

### **Created support for donations coming from the application.**

Ready - 
- [x] Loading donation data
- [x] Pay with Credit card
- [x] Pay with Native pay (Google/apple)

Under development - 
- [ ] Paypal

### **How to test ?** 

- Get a token/url from the app (visit browser stack and try to make a payment to receive the code)
- Visit the URL - http://localhost:3000/donation-payment/`TOKEN`
- Donate using CC
- Donate using Native pay
- Check you the Thank you page
